### PR TITLE
Add API doc for Airflow

### PR DESCRIPTION
**Pull request by yangaws**
*Issue #, if available:*

*Description of changes:*
1. Add API doc for Airflow
2. Fix API doc for RL

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -62,3 +62,12 @@ Amazon provides implementations of some common machine learning algortithms opti
     factorization_machines
     lda
     ntm
+
+SageMaker Workflow
```
Comment (laurenyu) [inline doc/index.rst]: would it make more sense to change this to just "Workflows"? (the workflow tools themselves aren't part of SageMaker)

File: doc/index.rst [hunk 2]
```diff
@@ -62,3 +62,12 @@ Amazon provides implementations of some common machine learning algortithms opti
...
+------------------
+SageMaker APIs to export configurations to create and manage Airflow workflows.
```
Comment (laurenyu) [inline doc/index.rst]: s/to create and manage/for creating and managing

Comment (yangaws) [reply doc/index.rst]: Updated!

File: doc/index.rst (hunk 1 above)
Comment (yangaws) [reply doc/index.rst]: Yep, this is better. 

Updated! Thanks!

**Pull request merged**